src/Tribe/Traits/Array_Access.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Array Access trait.
+ *
+ * @since TBD
+ *
+ * phpcs:disable WordPress.NamingConventions.ValidFunctionName
+ */
+
+declare( strict_types=1 );
+
+namespace Tribe\Traits;
+
+/**
+ * Trait Array_Access
+ *
+ * @since TBD
+ */
+trait Array_Access {
```

Comment:
This is awfully similar to [src/Tribe/Data.php](https://github.com/the-events-calendar/tribe-common/blob/master/src/Tribe/Data.php)

Reply:
Thanks! I actually ended up refactoring the `Data` class to use this trait. I wanted the ability to add `implements ArrayAccess` easily to any class that needs it.